### PR TITLE
Removed AlwaysIncluded field

### PR DESCRIPTION
**Pull request by janholinka**
#### Changelog notes 

```
* Removed AlwaysIncluded field
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: distributor-api-v1/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 18.06.2020
+
+* Removed **AlwaysIncluded** field from [Product](operations.md#product) ([37](https://github.com/MewsSystems/gitbook-distributor-guide/pull/37/files)).
```
[the remark this answers is not in the document]
Comment (koprivajakub) [reply distributor-api-v1/changelog.md:5]: I would add an explanation of why this should be done. Something like:
```
This flag does not have any meaning anymore. The API is returning it's value always false. In order to pre-select a product during a booking you can use `IncludedByDefault` flag.
```

[the remark this answers is not in the document]
Comment (janholinka) [reply distributor-api-v1/changelog.md:5]: @zdrazil Thanks I completely missed formatting. @koprivajakub I will add it there.

File: distributor-api-v1/changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 18.06.2020
+
+* Removed `AlwaysIncluded` field from [Product](operations.md#product) ([37](https://github.com/MewsSystems/gitbook-distributor-guide/pull/37/files)). This flag does not have any meaning anymore. The API is returning its value always equal to `false`. In order to pre-select a product during a booking, you can use `IncludedByDefault` flag.
```
Comment (vitasamek) [inline distributor-api-v1/changelog.md:5]: There is no flag in API. It's a boolean property.

Comment (vitasamek) [inline distributor-api-v1/changelog.md:5]: I would remove ` This flag does not have any meaning anymore. The API is returning its value always equal to `false`.` IMHO it says nothing to a developer.

**Comment by siroky** [comment]
@janholinka this is stale for more than a week, please action this.

**Pull request merged**